.circleci/config.yml

```diff
@@ -1,4 +1,17 @@
 version: 2.1
+parameters:
+  GHA_Event:
+    type: string
+    default: ""
+  GHA_Actor:
```

Comment:
Some of these aren't used, what are they for exactly?

Reply:
They are set by the circle ci plugin. The [article](https://circleci.com/blog/trigger-circleci-pipeline-github-action/) implies that they are required, but not entirely sure. Just added the ones they listed to be safe.